tests/security/atsec/drng_test_preparation.pm

```diff
@@ -0,0 +1,64 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Run 'DRNG test preparation' test case of ATSec test suite
+# Maintainer: xiaojing.liu <xiaojing.liu@suse.com>
+# Tags: poo#108485
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+
+    # Install the required packages
+    zypper_call('in gcc libopenssl-devel libgcrypt-devel');
+
+    # Complile gather_random_data
+    my $exe_file = 'gather_random_data';
+    assert_script_run('cd /usr/local/atsec');
+    assert_script_run("gcc -o $exe_file -lcrypto -lssl -lgcrypt gather_random_data.c");
+
+    # Prepare the test directory
+    assert_script_run('mkdir -p /root/eval/drng');
+
+    # Copy the executable file to test directory
+    assert_script_run('cp gather_random_data /root/eval/drng/');
+
+    # Run the test
+    assert_script_run('cd /root/eval/drng/');
+    assert_script_run("./$exe_file 2");
+
+    # Check result
+    my $result = script_output('ls -l | grep -v "total"');
+    my @lines = split(/\n/, $result);
+    record_info('ERROR', "Others files are generated\n$result", result => 'fail') if (scalar(@lines) != 4);
+
+    foreach my $line (@lines) {
+        my @items = split(/\s/, $line);
+        my $file_size = $items[4];
+        my $file_name = $items[-1];
+
+        # Skip gather_random_data
+        next if ($file_name eq $exe_file);
```

Comment:
Just suggestion: we can use `grep -v "total" | grep -v $exe_file` in lines `# Check result` to handle these 2 lines.